client/app/vps/modal/shortcut/reverse-dns/vps-reverse-dns.controller.js

```diff
@@ -46,8 +46,10 @@ class VpsReverseDnsCtrl {
                 if (data && data.state) {
                     switch (data.state) {
                     case "ERROR" :
-                        this.CloudMessage.error(this.$translate.instant("vps_configuration_reversedns_fail"))
-                        break;
+                        this.CloudMessage.error(this.$translate.instant("vps_configuration_reversedns_fail"));
+                        const messages = !_.isEmpty(data.messages) ? data.messages : [];
```

Comment:
C'est peut-être plus pratique de placer `(message) => this.CloudMessage.error(message.message || message))` entre les lignes 46 et 47 `if (data && data.state) { \n switch (data.state) {`.  Aussi probablement que ça n'a pas besoin d'être une constante.